src/popup/snooze.scss

```diff
@@ -254,53 +357,127 @@ li.option {
   color: #6E6E6E;
 }
 
-.option > .date {
-  flex: 0 0 65px;
+.rc-calendar {
+  font-family: 'Fira Sans';
+  font-size: 14px;
+  letter-spacing: -.69px;
```

Comment:
We seem to use these three lines a few times.  Should we pull them out into a mixin?  (Maybe in a future patch…)